### PR TITLE
D8CORE-562: Updates to Hero and Card templates

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Fixes and updates to templates hero + card
- Dependant on https://github.com/SU-SWS/decanter/pull/513

# Review By (Date)
- October 22nd

# Urgency
- Medium

# Steps to Test

1. Check out this branch
2. Do a full drupal site install of the stanford_profile profile
3. Review home page banner and cards
4. Remove all text from home page banner and review

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/pull/513

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: templates/components/hero/hero.html.twig [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * - hero_cta_label: The clickable text for the CTA link
+ * - hero_button_link: The href property of the button component.
+ * - hero_button_label: The text label of the button component.
+ */
+#}
+
+{#
+  Sanitize the Drupal
+#}
+{%- if hero_super_headline|render|striptags is empty -%}
```
Comment (pookmish) [inline templates/components/hero/hero.html.twig]: according to https://www.lullabot.com/articles/level-your-twiggery you always want to keep `<drupal-render-placeholder>` tags when doing this. Its also mentioned [here](https://www.drupal.org/project/drupal/issues/2672656#comment-13091978)

**Review by pookmish — CHANGES_REQUESTED** [review]
we should not strip `<drupal-render-placeholder>`

File: templates/components/hero/hero.html.twig (hunk 1 above)
Comment (sherakama) [reply templates/components/hero/hero.html.twig]: 💡 Yes, thank you.

**Comment by sherakama** [comment]
Ok, all dependant PRs have been merged and feedback on striptags applied. This is good for another pass.

**Review by pookmish — APPROVED** [review]
it works

**Pull request merged**